### PR TITLE
Added exporting to csv for all charts

### PR DESCRIPTION
**Pull request by frotein**
All charts pages now contain a button that returns a csv file of the current chart

oage

### CODE REVIEW

File: app/controllers/histogram_chart_controller.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 require 'histogram/array'  # enables Array#histogram
 require 'googlecharts'
```
Comment (MattColemanMan) [inline app/controllers/histogram_chart_controller.rb]: I had to comment these out to make the histogram page work on linux, are you sure its required?

File: app/controllers/histogram_chart_controller.rb [hunk 2]
```diff
@@ -48,13 +50,20 @@ def new
 		        end
 		        nd = (temp+ @freq).to_s
 		        st = temp.to_s
-		       	titles[ind] = (st + " to " + nd)
-		        @histoAr[ind] = freqNum
-		        temp += @freq
+		        title = (st + " to " + nd)
+            titles[ind] = title
```
Comment (MattColemanMan) [inline app/controllers/histogram_chart_controller.rb:52]: these titles are confusing, perhaps we should refactor and use better variable names.

File: app/controllers/histogram_chart_controller.rb (hunk 1 above)
Comment (frotein) [inline app/controllers/histogram_chart_controller.rb]: No I Dont believe so

File: app/views/json_url/show.html.erb [hunk 3]
```diff
@@ -6,14 +6,16 @@
   <%= label_tag(:to, "to: ") %>
   <%= number_field_tag(:to, @to, :min => @min, :max => @max) %>
   <%= submit_tag("Update") %>
-  
+  <% @returnc = false  %>
```
Comment (MattColemanMan) [inline app/views/json_url/show.html.erb:9]: what is returnc

Comment (MattColemanMan) [inline app/views/json_url/show.html.erb:9]: oh wait nevermind.

**Pull request merged**